Sources/MessagingInApp/Views/GistInlineInAppMessageView.swift

```diff
@@ -0,0 +1,242 @@
+import CioInternalCommon
+import Foundation
+import UIKit
+
+// Event listener for interactions and state changes to an inline inapp message that's rendered.
+public protocol GistInlineInAppMessageViewDelegate: AnyObject {
+    // After a message is finished rendering and the size may have changed.
+    func onMessageRendered(width: CGFloat, height: CGFloat)
```

Comment:
How about using `CGSize` as a parameter instead of two different parameters `width` and `height` ?

Reply:
I like the named parameters that label height and width. 

What benefit does `CGSize` give over this approach? I'm open to it if it would be better.